src/repos/FlyerRepo.ts

```diff
@@ -275,8 +278,26 @@ const createFlyer = async (
   return FlyerModel.create(newFlyer);
 };
 
+/**
+ * Delete a flyer
+ *
+ * @function
```

Comment:
Generally the `@function` is used in JSDoc when we want to specify an object is a function if it is unclear (e.g., if we did `let x = someFunc()` and someFunc() returns a function, we would mark x with @function. I don't think we need this here